### PR TITLE
Audit log migration query strings

### PR DESCRIPTION
**Pull request by anantailo**
Add audit log migration query strings

### CODE REVIEW

**Review by Smtih — COMMENTED** [review]
I think having these is useful, is it worth still providing the function wrappers to make these easy to use in a knex migration as well?

**Comment by anantailo** [comment]
> 0c58cdc

Added function wrappers in 0c58cdc

**Comment by anantailo** [comment]
> I think having these is useful, is it worth still providing the function wrappers to make these easy to use in a knex migration as well?

Added knex function wrappers in 0c58cdce9a639f80b733e8dcddbc8cbd20347f32

File: src/history-utils/createAuditLogMigration.ts [hunk 1]
```diff
@@ -0,0 +1,273 @@
+import Knex from "knex";
+
+export function createAuditLogMigration(): Knex.Migration {
+  return {
+    up,
+    down,
+  };
+}
```
Comment (Smtih) [inline src/history-utils/createAuditLogMigration.ts:8]: Think you'll want to add this file to the index.ts in this folder.

**Review by Smtih — CHANGES_REQUESTED** [review]

File: src/history-utils/createAuditLogMigration.ts (hunk 1 above)
Comment (anantailo) [reply src/history-utils/createAuditLogMigration.ts:8]: Exported and updated README.md in 5ce2da74962e1f4e00c49a2c5fe759f82c086b81

**Review by Smtih — APPROVED** [review]
LGTM

**Pull request merged**